src/Parallel/Parallel.jl

```diff
@@ -20,6 +20,7 @@ include("centrality/pagerank.jl")
 include("centrality/radiality.jl")
 include("centrality/stress.jl")
 include("distance.jl")
+# include("Traversals/Traversals.jl")
```

Comment:
remove?

Reply:
Eventually, yes. The idea is that most of the stuff in here will be moved into the main codebase under their respective submodules (the parallel traversals is now in `Traversals`; the parallel centrality will be in `Centrality`, etc.) As such, there's no urgency to doing this now since we're going to be removing lots of the stuff in here. 

In fact, I think that by the end we won't have a separate `Parallel` module at all.

Reply:
Removed.